src/Cupboard.Providers.MacOs/Cupboard.Providers.MacOs.csproj.DotSettings

```diff
@@ -0,0 +1,2 @@
+﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
+	<s:Boolean x:Key="/Default/CodeInspection/NamespaceProvider/NamespaceFoldersToSkip/=homebrew/@EntryIndexedValue">True</s:Boolean></wpf:ResourceDictionary>
```

Comment:
This might be better as an editorconfig 🤔

Reply:
Is that how it's currently implemented and I missed it?  Likely just me clicking buttons too fast during development trying to remove compiler warnings.  I can remove this file